scripts/dpdk_setup_ports.py

```diff
@@ -1223,7 +1223,8 @@ def _get_wanted_interfaces(self, input_interfaces, get_macs = True):
         return wanted_interfaces
 
     def do_create(self):
-        dpdk_nic_bind.show_table(True) # get the info 
+        if not map_driver.args.no_prompt:
+            dpdk_nic_bind.show_table(True)  # get the info
```

Comment:
This line is to get driver info ('MAC' and 'TRex_Driver'). So, this line shouldn't be skipped.